backend/server/dissectors/dissected_request.go

```diff
@@ -117,7 +116,11 @@ func (m *DissectedRequest) FromURL(key string) *Coercable {
 	if m.Error != nil {
 		return makeErrCoercable(m)
 	}
-	value, found := m.urlValues[key]
+	value := chi.URLParam(m.Request, key)
```

Comment:
Just spit balling here, but maybe instead of passing in a map, or using chi directly (thus tying this to a chi-only solution), we could instead take in a function to get a parameter, given a key?

Reply:
oh yeah that would be better - I think I'm going to try to go back to the map of keys and values, which will change this code back to the way it was